components/collective-page/BudgetStats.js

```diff
@@ -0,0 +1,142 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Calendar } from '@styled-icons/feather/Calendar';
+import { Expand } from '@styled-icons/ionicons-solid/Expand';
+import { ShowChart } from '@styled-icons/material/ShowChart';
+import { FormattedMessage } from 'react-intl';
+import styled, { css } from 'styled-components';
+
+import { CollectiveType } from '../../lib/constants/collectives';
+import { formatCurrency, getCurrencySymbol } from '../../lib/currency-utils';
+
+import Container from '../Container';
+import DefinedTerm, { Terms } from '../DefinedTerm';
+import FormattedMoneyAmount from '../FormattedMoneyAmount';
+import { Box } from '../Grid';
+import { P } from '../Text';
+
+const StatTitle = props => (
+  <P fontSize="11px" lineHeight="12px" fontWeight="500" color="black.700" textTransform="uppercase" mb={2} {...props} />
+);
+
+const StatAmount = ({ amount, ...props }) => (
+  <P fontSize="16px" lineHeight="24px" color="black.700">
+    {/* Pass null instead of 0 to make sure we display `--.--` */}
+    <FormattedMoneyAmount amount={amount || null} {...props} />
+  </P>
+);
+
+StatAmount.propTypes = {
+  amount: PropTypes.number,
+};
+
+const StatContainer = styled.div`
+  flex: 1;
+  padding: 16px 32px;
+
+  svg {
+    margin-right: 5px;
+    vertical-align: bottom;
+  }
+
+  ${props =>
+    props.$isMain &&
+    css`
+      background: #f7f8fa;
+    `}
+
+  ${props =>
+    !props.$isFirst &&
+    css`
+      border-top: 1px solid #dcdee0;
+    `}
+`;
+
+const BudgetStats = ({ collective, stats }) => {
+  const monthlyRecurring =
+    (stats.activeRecurringContributions?.monthly || 0) + (stats.activeRecurringContributions?.yearly || 0) / 12;
+  const isFund = collective.type === CollectiveType.FUND;
+  const isProject = collective.type === CollectiveType.PROJECT;
+
+  return (
+    <Fragment>
+      <StatContainer data-cy="budgetSection-today-balance" $isFirst>
+        <StatTitle>
+          <Container display="inline-block" fontSize="11px" mr="5px" fontWeight="500" width="12px" textAlign="center">
+            {getCurrencySymbol(collective.currency)}
+          </Container>
+          <FormattedMessage id="CollectivePage.SectionBudget.Balance" defaultMessage="Today’s balance" />
+        </StatTitle>
+        <StatAmount amount={stats.balance} currency={collective.currency} />
+      </StatContainer>
+      <StatContainer>
+        <StatTitle>
+          <ShowChart size="12px" />
+          <FormattedMessage id="budgetSection-raised" defaultMessage="Total raised" />
```

Comment:
@znarf Total raised and total disbursed are now displayed for everyone. I also updated the copy/styles and added icons to match the design.